syntax = "proto3";

package appointment;

service AppointmentService {
  rpc GetAppointmentById (AppointmentIdRequest) returns (AppointmentResponse);
}

message AppointmentIdRequest {
  int32 appointment_id = 1;
}

message AppointmentResponse {
  bool found = 1;
  int32 appointment_id = 2;
  int32 patient_id = 3;
  string doctor_id = 4;
  string status = 5;
  string description = 6;
  string date = 7;
  string started_time = 8;
}
